Makefile.core.mk

```diff
@@ -29,21 +29,23 @@ lint: clean_public gen_nominify
 serve: build
 	@hugo serve --baseURL "http://${ISTIO_SERVE_DOMAIN}:1313/" --bind 0.0.0.0 --disableFastRender
 
-install:
+# used by netlify.com when building the site. The tool versions should correspond
+# to what is included in the tools repo in docker/build-tools/Dockerfile.
+netlify_install:
 	@npm init -y
-	@npm install -g \
-	    sass \
-	    sass-lint \
-	    typescript \
-	    tslint \
-	    markdown-spellcheck \
-	    svgstore-cli \
-	    svgo
-	@npm install --save-dev @babel/core @babel/cli @babel/preset-env
-	@npm install --save-dev babel-preset-minify
-	@npm install --save @babel/polyfill
-
-netlify: install
+	@npm install --production --global \
+	    sass@v1.22.10 \
+	    typescript@v3.5.3 \
+	    svgstore-cli@v1.3.1 \
+		@babel/core@v7.5.5 \
+		@babel/cli@v7.5.5 \
+		@babel/preset-env@v7.5.5
+	@npm install --production --save-dev \
+		babel-preset-minify@v0.5.1
+	@npm install --save-dev \
```

Comment:
this is inconsistent with the container implementation(the --save-dev which is probably needed according to the documentation) (and I htink also the version which IIRC is 7.5.0 for polyfill.)

Reply:
I don't see the inconsistencies. See https://github.com/istio/tools/blob/8a495d7e5de49ee394a33970492ca876a05ade77/docker/build-tools/Dockerfile#L141

Am I missing something?

Reply:
BTW, When you install the polyfill stuff, it prints a message saying it is deprecated. But when I remove it, nothing works...

Reply:
right - I reviewed quickly and got version numbers mixed up in my intermediate-term memory (which is unreliable;) . I saw that deprecation problem with polyfill scroll by.  I'll see if we can sort out the replacement.  NODEJS FTW.